regen/warnings.pl

```diff
@@ -912,7 +915,8 @@ =head1 DESCRIPTION
 The C<warnings> pragma gives control over which warnings are enabled in
 which parts of a Perl program.  It's a more flexible alternative for
 both the command line flag B<-w> and the equivalent Perl variable,
-C<$^W>.
+C<$^W>.  This pragma is enabled automatically by a
```

Comment:
This should probably be clear that it does the equivalent of `use warnings` or `use warnings 'all'`, since there are a lot of ways to use the pragma.

Reply:
Good idea.